glustercli/cmd/volume-create.go

```diff
@@ -47,10 +47,11 @@ func init() {
 	volumeCreateCmd.Flags().BoolVar(&flagCreateForce, "force", false, "Force")
 	volumeCreateCmd.Flags().StringSliceVar(&flagCreateVolumeOptions, "options", []string{},
 		"Volume options in the format option:value,option:value")
-	volumeCreateCmd.Flags().BoolVar(&flagCreateAdvOpts, "advanced", false, "Allow advanced options")
```

Comment:
Wait, did you delete these ?

Reply:
my bad got deleted while resolving the conflict. updated now